src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -138,6 +138,16 @@
     text: az appconfig kv set --connection-string Endpoint=https://contoso.azconfig.io;Id=xxx;Secret=xxx --key color --value red --tags key1=value1 key2=value2
 """
 
+helps['appconfig kv set-keyvault'] = """
```

Comment:
From my understanding, it is a little similar with `az appconfig kv set` command. Why not use `az appconfig kv set` to set keyvault value?

Reply:
To set key vault reference, we need to provide users with different arguments than the set kv command. This also aligns with the portal experience, where set kv and set key vault are different.

---
In reply to: [361830243](https://github.com/Azure/azure-cli/pull/11571#discussion_r361830243) [](ancestors = 361830243)